packages/snaps-controllers/src/snaps/registry/json.test.ts

```diff
@@ -37,11 +38,23 @@ const MOCK_DATABASE: SnapsRegistryDatabase = {
   ],
 };
 
+const MOCK_SIGNATURE =
+  '0x3045022100cc049732d4cc8b888162b8b998c9beefcc1de8c4489594c7c504d4aa031223af02206835834ba2b7bec45b2a9e1e72312d69445d6a0f5590a2408bb13931a14f6c8a';
+const MOCK_PUBLIC_KEY =
+  '0x034ca27b046507d1a9997bddc991b56d96b93d4adac3a96dfe01ce450bfb661455';
+const MOCK_SIGNATURE_FILE = {
+  signature: MOCK_SIGNATURE,
+  publicKey: MOCK_PUBLIC_KEY,
```

Comment:
Why do we need the `MOCK_PUBLIC_KEY` here? 🤔

Reply:
That's the format used in MetaMask/snaps-registry#18. It's not used, but seems better to mock with "real" data.